.github/workflows/ci.yml

```
@@ -26,15 +28,18 @@ jobs:
           version: ${{ env.PNPM_VERSION }}
+          run_install: false
 
       - name: Get pnpm store directory
+        id: pnpm-cache
         shell: bash
-        run: echo "STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
+        run: |
+          echo "STORE_PATH=$(pnpm store path)" >> $GITHUB_OUTPUT
 
       - name: Setup pnpm cache
         uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)